### ISSUE TITLE
Structure Alt-Text output accoding to headings

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#267]
This should be "article" style with keyboard/screen reader navigable headers

**Comment by JakeWags** [comment]
Depends on [alt text #30](https://github.com/visdesignlab/upset-alt-txt-gen/issues/30)
